build.gradle

```
@@ -0,0 +1,5 @@
+plugins {
+    // apply false : 전체 하위 프로젝트에 플러그인을 적용하고 싶지만 rootProject에서는 적용하고 싶지 않을 경우 사용
+    id 'org.springframework.boot' version '3.1.3' apply false
+    id 'io.spring.dependency-management' version '1.1.3' apply false
```

알파벳 순서를 맞추면 나중에 새로운 걸 추가할 때, 고민없이 추가할 수 있는 장점이 있죠. :-)
```suggestion
    id 'io.spring.dependency-management' version '1.1.3' apply false
    id 'org.springframework.boot' version '3.1.3' apply false
```